support pushing draft commit to BSR when draft param is true (#21)
bufbuild/buf-push-action: README.md

    - uses: bufbuild/buf-push-action@v1
      with:
        buf_token: ${{ secrets.BUF_TOKEN }}
        enable_draft: true
```

> With this configuration, the `buf` CLI pushes the [configured module][buf-yaml] to the BSR using 
> a Buf API token to authenticate with the [Buf Schema Registry][bsr] (BSR), upon a pull request 
> [opened, synchronize, or reopened][github-workflow], and will push as a draft when the triggering 
> branch is not `main`.

Reviewer: ```suggestion
With this configuration, upon a pull request [opened, synchronize, or reopened][github-workflow] 
the `buf` CLI pushes the [configured module][buf-yaml] to the BSR using the provided to 
authenticate the request. When the triggering branch is not `main`, the commit will be pushed 
as a [draft][<put-draft-docs-link-here>].
```

Some run on sentences here, I think this captures it? Still needs a link to the draft docs.

Author: updated

## Configuration

| Parameter      | Description                                                                                           | Required | Default                             |
|:---------------|:------------------------------------------------------------------------------------------------------|:---------|:------------------------------------|
| `buf_token`    | The [Buf authentication token][buf-token] used for private [Buf inputs][input]                        | ✅        | [`${{github.token}}`][github-token] |
| `input`        | The path of the [input] you want to push to BSR as a module                                           |          | `.`                                 |

> | `enable_draft` | Enable to push the BSR commit as draft when the git branch that triggering the workflow is not `main` |          |                                     |

Reviewer: ```suggestion
| `draft` | Indicates if non-`main` workflows should be ignored or push to the BSR as draft commit  |          |                                     |
```
With the param being a boolean, I don't think we need a `enable_` prefix? Do we do that in other github actions?

If we keep this as just `drafts`, we can later extend it to check if its a boolean or a string, and in the string case only push if it matches.

Author: updated, I think its actually 

> Indicates if non-`main` workflows should push to the BSR as draft commit or main commit
